### PR TITLE
Hide Apps tab for Admin (#2634)

### LINKED ISSUE #2634: No apps tab required for Admin in Manage Groups.
### What is the expected behaviour?
There should be no apps tab required for Admin because admin will always have read/write permission for all the apps.

### What is the current behaviour?
Apps tab present in manage groups for admins.

### How to reproduce the issue?

1. Go to manage groups.
2. Go to admins & observe.

### Screenshots or Screencast
https://drive.google.com/file/d/1EoK-lEcAGtuv0sNW04u53QZ9BpSxj0Ge/view?usp=sharing

### PR DESCRIPTION
**Pull request by daniloff200**
closes #2634

### CODE REVIEW

**Comment by gsmithun4** [comment]
@daniloff200 Can you please fix the lint errors
https://github.com/ToolJet/ToolJet/actions/runs/3169781786/jobs/5161925145

**Comment by daniloff200** [comment]
@gsmithun4  hi! Yeah, I've updated the PR

**Comment by daniloff200** [comment]
@gsmithun4  I'm sorry, therer was one small issue in pipeline. Could you re-run it again, please?

**Comment by daniloff200** [comment]
Can this one be reviewed?

**Review by gsmithun4 — CHANGES_REQUESTED** [review]
Shall we make the user tab active for admin group?
Screenshot 2022-10-06 at 7 03 03 PM

**Comment by daniloff200** [comment]
@gsmithun4  I can do that : )

**Comment by daniloff200** [comment]
@gsmithun4  done :)
Can you look at this, please?

**Comment by Nandinisaha13** [comment]
Hi @daniloff200 , When I clicked on Admin on manage groups the app crashes. Attaching the video for reference!
[hide_app_bug.webm](https://user-images.githubusercontent.com/41302917/194475973-c76dd60c-a140-4ffc-91e9-ab04e67100fe.webm)

**Comment by daniloff200** [comment]
hi @Nandinisaha13 
I've made one small change, of you're able to redeploy the app, could you check that, please?

**Comment by Nandinisaha13** [comment]
We deployed the new changes then only this is crashing otherwise it was atleast working before the last change you made! Please have a look! @daniloff200

**Comment by daniloff200** [comment]
Yeah, I saw that, I've made the changes to prevent that

**Comment by Nandinisaha13** [comment]
Got it! @daniloff200 ! Will check & let you know! Thank You

File: frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx [hunk 1]
```diff
@@ -48,6 +48,10 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
 
   fetchGroupPermission = (groupPermissionId) => {
     groupPermissionService.getGroup(groupPermissionId).then((data) => {
+      if (data?.group !== 'admin') {
+        this.setState({ currentTab: 'users' });
+      }
+
       this.setState({
         groupPermission: data,
```
Comment (gsmithun4) [inline frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx:56]:
```suggestion
      this.setState({
        currentTab: data?.group !== 'admin' ? 'users' : 'apps',
        groupPermission: data,
```

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx (hunk 1 above)
Comment (daniloff200) [reply frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx]: Thanks for suggestion! I've updated my code

File: frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx [hunk 2]
```diff
@@ -50,6 +50,7 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
     groupPermissionService.getGroup(groupPermissionId).then((data) => {
       this.setState({
         groupPermission: data,
+        currentTab: data?.group !== 'admin' ? 'users' : 'apps',
```
Comment (gsmithun4) [inline frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx:53]: looks like we should reverse the condition, should be
```
currentTab: data?.group !== 'admin' ? 'apps' : 'users',
```

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx (hunk 2 above)
Comment (daniloff200) [reply frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx]: done

**Comment by Nandinisaha13** [comment]
Hy @daniloff200 While initially load I can see the Apps tab still visible, one solution could be if you can show the loader on the page so that way only loader is being dis[inital_load.webm](https://user-images.githubusercontent.com/41302917/194824457-39d4d71f-6926-47f7-8572-ceedf398aa18.webm)yed even on initial load. Attaching screencast for the same.

File: frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx [hunk 3]
```diff
@@ -355,15 +354,22 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
 
           <div className="page-body">
             <div className="container-xl">
+              {isLoadingGroup && (
```
Comment (gsmithun4) [inline frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx:357]: Spinner is not used as page loader
Please try something like this 
https://github.com/ToolJet/ToolJet/blob/develop/frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx#L447

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx [hunk 4]
```diff
@@ -355,15 +354,22 @@ class ManageGroupPermissionResourcesComponent extends React.Component {
...
+                <div className="row">
+                  <div className="skeleton-line w-10 col mx-3"></div>
+                </div>
+              )}
```
Comment (gsmithun4) [inline frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx:361]:
```suggestion
              {isLoadingGroup ? <Loader /> : (<div className="card">//Rest of the components// </div>)}
```
Shall we re-use the Loader `src/ManageSSO/Loader.jsx`

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx (hunk 4 above)
Comment (daniloff200) [reply frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx]: done

**Comment by daniloff200** [comment]
Fixed a lint issue, not sure, how it happened, bur was fixed during rebase with the develop branch

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx [hunk 5]
```diff
@@ -7,6 +7,9 @@ import { toast } from 'react-hot-toast';
 import { Link } from 'react-router-dom';
 import { withTranslation } from 'react-i18next';
 
+import Spinner from '@/_ui/Spinner';
```
[the remark this answers is not in the document]
Comment (daniloff200) [reply frontend/src/ManageGroupPermissionResources/ManageGroupPermissionResources.jsx]: done

**Comment by gsmithun4** [comment]
please revert the submodule `hacktoberfest-swag-list`

**Review by gsmithun4 — APPROVED** [review]

**Pull request merged**